lalrpop-test/src/main.rs

```diff
@@ -142,6 +142,26 @@ fn expr_intern_tok_test_err() {
 }
 
 #[test]
+fn expr_intern_tok_display_err() {
+    let expr = "(1+\n(2++3))";
+    let result = expr_intern_tok::parse_Expr(1, expr);
+    let err : lalrpop_util::ParseError<usize, (usize, &str),()>
+            = result.unwrap_err();
+
+    // The problem is that (usize,&str) and () do not implement fmt::Display,
+    // so neither does the ParseError.
+    // We can fix that by rewriting them to something that has fmt::Display
+    let disp = err.map_tok(|(_,t)|t).map_err(|_| "error");
+    assert!(disp.to_string().contains("Unrecognized token +"));
```

Comment:
It's not due to your code, but could we changed the message here to use backticks to "quote" the token? i.e.:

```
Unrecognized token `+`
```